tests/testthat/test-compactify.R

```diff
@@ -0,0 +1,62 @@
+library(epiprocess)
+library(data.table)
+library(dplyr)
+
+dt <- archive_cases_dv$DT
```

Comment:
Expecting an uncompactified `archive_cases_dv$DT` here will probably break the tests if `archive_cases_dv` is re-built down the line with the default `compactify` setting.  Maybe update the `archive_cases_dv` generation code to use `compactify=FALSE`, or use `TRUE` and make the code below insert some LOCF rows or something?

Reply:
I noticed that the dataset stored in R as an rda in data is different from what happens if I were to run the R code. It works for now; however, I will need to update it when it gets fixed to allow for LOCF to be used.

Reply:
Sorry, I didn't see that you were working on this and set something up for this.  It rebuilds the archive from a DT whenever the user first requests it, and uses compactify=FALSE to keep the current tests running.